backend/Dockerfile

```diff
@@ -1,18 +1,22 @@
-FROM ubuntu:16.04
+FROM python:3.8
 
 EXPOSE 5000
 
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-RUN apt-get update -y && apt-get install -y python3-pip libpq-dev && apt-get autoremove
+RUN apt-get update -y && apt-get install -y libpq-dev && apt-get autoremove
 
-RUN pip3 install --upgrade pip
+RUN pip install --upgrade pip
+RUN apt-get update
```

Comment:
What's the reason for updating apt registry if you're not using it below?

Reply:
Removed.